### ISSUE TITLE
Walkthrough for digital ocean server (i.e. private VPS)

### ISSUE THREAD

**Issue by dom-devel** [wooey/Wooey#168]
I'm having some problems setting up Wooey on my own DO server. I found a support ticket that mentioned someone was thinking about writing up a guide for DO, but I don't think anything ever appeared. 

The project isn't local, so all the problems have come from trying to get my app to appear publicly on port 80. (And once that is working, then move to HTTPS only).

I began following [this tutorial](http://uwsgi-docs.readthedocs.io/en/latest/tutorials/Django_and_nginx.html), and successfully installed nginx (from my own poor understanding this appeared to be the correct choice for forcing HTTPS only). Pointed the nginx upstream to my Wooey server, however I only get 400 errors. 

```
[18/Dec/2016 12:46:31] code 400, message Bad request syntax ('\x00\x10\x03\x00\x0c\x00QUERY_STRING\x00\x00\x0e\x00REQUEST_METHOD\x03\x00GET\x0c\x00CONTENT_TYPE\x00\x00\x0e\x00CONTENT_LENGTH\x00\x00\x0b\x00REQUEST_URI\x01\x00/\t\x00PATH_INFO\x01\x00/\r\x00DOCUMENT_ROOT\x15\x00/usr/share/nginx/html\x0f\x00SERVER_PROTOCOL\x08\x00HTTP/1.1\x0e\x00REQUEST_SCHEME\x04\x00http\x0b\x00REMOTE_ADDR\x0f\x00my_home_ip\x0b\x00REMOTE_PORT\x05\x0049279\x0b\x00SERVER_PORT\x02\x0080\x0b\x00SERVER_NAME\x0c\x00my_server_ip\t\x00HTTP_HOST\x0c\x00my_server_ip\x0f\x00HTTP_CONNECTION')
[DOCUMENT_ROOT2/usr/share/nginx/htmlTSERVER_PROTOCOLHTTP/1.1GREQUEST_SCHEME♦ httpNREMOTE_ADDR my_home_ip REMOTE_PORT♣ 49279 SERVER_PORT 80 SERVER_NAME my_server_name HTTP_HOST my_server_name HTTP_CONNECTION" 400 -
```

Is there any documentation on how to get this set-up on DO or your own VPS? For a beginner like myself I'm struggling to get it up and running in production.

**Comment by dom-devel** [comment]
Here is a running tutorial. I have successfully got it running on port 80. Those more knowledgeable may want to correct some things in this tutorial.

To begin with install nginx, uwsgi and wooey.

I ran the Wooey install in my home directory and so ended up with manage.py in `/home/user/project_name`.

I then installed stable nginx (currently 1.10.1), you can find instructions [here](https://www.digitalocean.com/community/tutorials/how-to-install-the-latest-version-of-nginx-on-ubuntu-12-10).

I kept running into problems with UFW blocking non-standard ports so ended up following the iptables rules provided by Linode over [here](https://www.linode.com/docs/security/firewalls/control-network-traffic-with-iptables).

I then basically followed [this tutorial](http://uwsgi-docs.readthedocs.io/en/latest/tutorials/Django_and_nginx.html), however I did a couple things differently. First I didn't install in a virtualenv because I'd already installed Wooey in my main env.

Then I managed the follow the steps below with no problems:

1. Basic test -- Because I was having trouble using non-standard ports, I replaced 8000 with 80 and ran the commands as root
2. Test your Django project -- In order to get this to work correctly I found I had to call uwsgi from the main folder where manage.py lives, in my case that was /home/user/ProjectName (I have no idea why, but otherwise it won't find the Wooey project correctly)
3. Deploying static files -- Have ignored this so far
4. nginx and uWSGI and test.py -- I got this to work on port 80 again because of non-standard port problems

At this point I tried to just use a network socket, because conceptually it was easier for me to understand.  (Crucially I am running this from /home/user/ProjectName)

uwsgi --socket 127.0.0.1:8000 --wsgi-file ProjectName/wsgi.py --chmod-socket=666

I then created a file in the wooey directory to be used as socket and switched to a file socket (following the step _"Using Unix sockets instead of ports"_:

uwsgi --socket ProjectName/django.sock --wsgi-file ProjectName/test.py --chmod-socket=666

**Comment by Chris7** [comment]
Thanks for putting this together! Would you be able to put these instructions as a pull request to the documentation?

**Comment by Chris7** [comment]
Implemented as #170! Thanks again.
